keras_nlp/src/utils/transformers/safetensor_utils.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+try:
+    import safetensors
+except ImportError:
+    safetensors = None
+
+
+def set_keras_weight(
+    safetensor_files,
+    safetensor_config,
+    keras_variable,
+    hf_weight_key,
+    hook_fn=None,
+):
+    if safetensors is None:
+        raise ImportError(
+            "`set_keras_weights()` requires the `safetensors` package. "
```

Comment:
Reword this to not reference an internal function. E.g. "Converting from the huggingface/transformers model format requires the `safetensors` package."